### PR TITLE
Add use case guide for HTTP routing with Traefik

### PR DESCRIPTION
**Pull request by mikesir87**
## Description

Adds a new use case guide for HTTP routing with Traefik

## Reviews

- [ ] Technical review
- [x] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/guides/use-case/traefik.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- [Node.js](https://nodejs.org/en/download/package-manager) and [yarn](https://yarnpkg.com/)
+- Basic of Docker
+
+## Using Traefik with Docker
+
+One of the unique features of Traefik is its ability to be configured in many ways. When using the Docker provider, Traefik gets its configuration from other running containers using [labels](https://docs.docker.com/config/labels-custom-metadata/). Traefik will watch engine events (for container starts and stops), extract the labels, and update its configuration.
+
+The two most common Traefik-monitored labels will be ([view the full list here](https://doc.traefik.io/traefik/routing/providers/docker/)):
```
Comment (sarahsanders-docker) [inline content/guides/use-case/traefik.md:34]: Not a blocker, just a small thing I noticed. the double ()s cause an extra space in the deploy preview 
image

File: content/guides/use-case/traefik.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    ```
+
+    Once the container starts, open your browser to http://welcome.localhost. You should see a “Welcome to Docker” website.
+
+
+## Using Traefik in development
+
+Now that you’ve experienced Traefik, it’s time to try using it in a development environment. In this example, you will use a sample application that has a split frontend and backend. This app stack has the following configuration:
+
+1. All requests to /api to go to the API service
```
Comment (sarahsanders-docker) [inline content/guides/use-case/traefik.md:76]: should /api be in code? `/api` 
same with Step 3: `db.localhost`

**Review by sarahsanders-docker — DISMISSED** [review]

File: content/guides/use-case/traefik.md (hunk 1 above)
Comment (mikesir87) [reply content/guides/use-case/traefik.md:34]: Huh! Any idea how to resolve that seeing I have no space in the text itself? Could always just rewrite that segment to take it out of the ()s too.

Maybe something like this?

```suggestion
While there are [many Traefik-monitored labels](https://doc.traefik.io/traefik/routing/providers/docker/), the two most common will be:
```

Comment (sarahsanders-docker) [reply content/guides/use-case/traefik.md:34]: I like that fix!

**Comment by dvdksn** [comment]
Rebased the PR and updated front matter metadata, ptal @mikesir87

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**